pkg/enum/sharepoint.go

```
@@ -0,0 +1,554 @@
+			if !ok || len(fields) == 0 {
+				continue
+			}
+
+			// Render fields as key: value lines.
+			var sb strings.Builder
+			sb.WriteString("List: " + listName + "\n")
+			sb.WriteString("Site: " + siteName + "\n")
+			sb.WriteString("---\n")
+
+			for k, v := range fields {
+				sb.WriteString(fmt.Sprintf("%s: %v\n", k, v))
```

golangci-lint (2.12.2)

[error] QF1012: Use fmt.Fprintf(...) instead of WriteString(fmt.Sprintf(...))

(staticcheck)